Moved display of smartblock files that meet criteria to update.phtml

Robbt opened the pull request:
Basically I recreated what originally existed in the smart-block-criteria.phtml view and put it at the top of the track listing so that it will display with update.phtml and be updated when the criteria is changed and saved.

I'm not 100% happy with how this changes the UI but it is definitely the easiest option at this point and avoids touching the javascript to try to update it. Any suggestions for UI improvements are encouraged.  It might make sense to color it with CSS to make it more distinct but otherwise I think that this reimplements the functionality requested in #366

hairmare commented on airtime_mvc/application/views/scripts/form/smart-block-criteria.phtml:
```diff
@@ -120,7 +120,9 @@
 
-        <?php if ($this->showPoolCount) { ?>
+        <?php
+        // this will never run due to refactoring of playlist controller. it is replaced with code in update.phtml
```
Can we remove this whole block then? Since `$this->showPoolCount` is long gone this is just dead code and we can get rid of the whole div without any side effects.

hairmare commented on airtime_mvc/application/views/scripts/playlist/update.phtml:
```diff
@@ -2,6 +2,35 @@
+                    </span>
+    <span class='checked-icon sp-checked-icon' id='sp_pool_count_icon'></span>
+    <?php
+    } else if ($this->poolCount == 1) {
```
You might want to consider using [plural support in zf1](https://framework.zend.com/manual/1.11/en/zend.translate.plurals.html) for this. You should be able to use the adapter as follows:

```php
echo $this->getTranslator()->plural("file meets the criteria", "files meets the criteria", $this->poolCount);
```

This would also shift the burden of knowing how to handle the zero case to the translators (TIL that not all languages handle the zero case like English).

Robbt replied:
Looking into this. Your suggestion didn't work as getTranslator doesn't exist as a function. I'm trying to figure out if I should create a new Translator object to pull in plurals support. The only mention of Zend_Translate I see is in the rest RouteController.php file at this point.

hairmare commented on airtime_mvc/application/views/scripts/playlist/update.phtml:
```diff
@@ -2,6 +2,35 @@
 $isStaticSmartBlock = ($this->obj instanceof Application_Model_Block && $this->obj->isStatic());
 $isPlaylist = ($this->obj instanceof Application_Model_Playlist);
+if ($this->poolCount) { ?>
+<div class='sp_text_font sp_text_font_bold'>
```
Some CSS might have gone missing, at least the text seems to have very low contrast. Most likely this got kicked during a redesign and never had it's style updated.

![image](https://user-images.githubusercontent.com/116588/34355055-e7250240-ea32-11e7-8f22-03cce54616a3.png)

We might want to consider cleaning this up for more contrast. I won't hold off on merging over this though.

Robbt commented:
I did a commit with ngettext since the Zend_Translate object appeared to be missing. I also deleted some of the deadcode and fixed the logic so that it would display a pool count only on updates and the no files match message.
